index.html

```diff
@@ -177,31 +191,37 @@ <h1 class="big-title big-title-about">About me</h1>
                     <img src="images/Rectangle 55.png" alt="Skills about Reactangle">
                     <h5>Language</h5>
                 </div>
-                <p>JavaScript</p>
-                <p>Ruby</p>
-                <p>HTML</p>
-                <p>CSS</p>
+                <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
```

Comment:
- [ ] You did a great job working on the previous reviewer's suggestions! 🚀 However, I think that what the reviewer meant was that there's an "outer" list: (Languages, Frameworks, Skills) and inside each item of that list, there's another one listing its contents, something like this:

```html
<ul>
  <li>
    <h4>Languages</h4>
    <ul>
      <li>JavaScript</li>
      <li>Ruby</li>
      <li>HTML</li>
      <li>CSS</li>
    </ul>
  </li>
  <li>
    <h4>Frameworks</h4>
    ...
  </li>
  <li>
    <h4>Skills</h4>
    ...
  </li>
</ul>
```

Please consider applying this change! 🤓 🔧